app/view_models/account_show.rb

```diff
@@ -61,6 +62,14 @@ def totp_partial
     end
   end
 
+  def piv_cac_partial
+    if decorated_user.piv_cac_enabled?
+      'accounts/actions/disable_piv_cac'
```

Comment:
Code Climate says this is not tested. Just adding a note here to address this during the subsequent PRs.

Reply:
I expect this to get covered when I write feature tests.